dataactvalidator/interfaces/validatorStagingInterface.py

```diff
@@ -1,77 +1,52 @@
 from dataactcore.models.stagingInterface import StagingInterface
-from dataactcore.models.stagingModels import FieldNameMap
-from dataactcore.utils.responseException import ResponseException
-from dataactcore.utils.statusCode import StatusCode
-from dataactvalidator.interfaces.validatorJobTrackerInterface import ValidatorJobTrackerInterface
-from sqlalchemy import MetaData, Table
-from sqlalchemy.exc import NoSuchTableError
+from dataactcore.models.stagingModels import Appropriation, ObjectClassProgramActivity, AwardFinancial, AwardFinancialAssistance
 
 
 class ValidatorStagingInterface(StagingInterface):
     """ Manages all interaction with the staging database """
 
-    def dropTable(self,table):
+    def insertSubmissionRecordByFileType(self, record, fileType):
+        """Insert a submitted file row into staging.
+
+        Args:
+            record: record to insert (Dict)
+            submissionId: submissionId associated with this record (int)
+            fileType: originating file type of the record (string)
         """
+        if fileType == "award":
+            rec = AwardFinancialAssistance(**record)
+        elif fileType == "award_financial":
+            rec = AwardFinancial(**record)
+        elif fileType == "appropriations":
+            rec = Appropriation(**record)
+        elif fileType == "program_activity":
+            rec = ObjectClassProgramActivity(**record)
+        self.session.add(rec)
+        self.session.commit()
+
+    def getSubmissionRecordsByFileType(self, submissionId, fileType):
+        """Return records for a specific submission and file type.
 
         Args:
-            table: Table to be dropped
+            submissionId: the submission to retrieve records for (int)
+            fileType: the file type to pull (string)
 
         Returns:
-            True if successful
+            Query
         """
-
-        metadata = MetaData()
-        stagingTable = Table(table, metadata, autoload_with=self.engine)
-        stagingTable.drop(bind=self.engine)
-
-    def tableExists(self,table):
-        """ True if table exists, false otherwise """
-        return self.engine.dialect.has_table(self.engine.connect(),table)
-
-    def countRows(self,table):
-        """ Returns number of rows in the specified table """
-        metadata = MetaData()
-        try:
-            stagingTable = Table(table, metadata, autoload_with=self.engine)
-        except NoSuchTableError:
-            return 0
-        rows = self.session.query(stagingTable).count()
-        self.session.close()
-        return rows
```

Comment:
The only place this was being used was in the unit tests. To replace it, I used one of the new methods that get staging records by submission and got the count on it (see https://github.com/fedspendingtransparency/data-act-broker-backend/pull/78/files).